framework/src/controls/PIDTransientControl.C

```diff
@@ -170,5 +171,6 @@ PIDTransientControl::execute()
 
     // Keep track of the previous delta for integral windup control
     _old_delta = delta;
+    _old_delta_unaffected = delta;
```

Comment:
this is being updated on failed solves. So it's not correct.

Reply:
hi @GiudGiud, I think it is okay. The checks were failing because I had not updated a gold csv file before. Now all checks have passed.

Reply:
I dont think it's ok. 
You can add a test using the Failing problem and a custom time stepper with FunctionDT

then you make the time stepper double one time step and the FailingProblem (a test object) fail that step and cut it back in half.
The output of the test should be the same as with a regular non-failing problem

Reply:
hi @GiudGiud, I am leaving INL tomorrow morning and I'm afraid I won't be able to do it within this time. The derivate control is working with the adjustment so far with all the test inputs already implemented, it'd be great if you could consider having the improvement I've made.